src/test/resources/com/puppycrawl/tools/checkstyle/checks/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -102,4 +102,17 @@ void foo() {
 	private String notAUnicodeEscaped2 = "\\\\u1234";
 
 	private String onlyEscaped = "\\\u1234";
+
+	private String controlC0 = ""
+			+ "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\n\u000b\u000c\r\u000e\u000f"
```

Comment:
It seems that it is impossible to use it in string literal without compiler error. We cannot split a string literal into two lines. It will surely produce an `illegal line end in character literal` error.
smth like
```
\\ there will be a line feed at the end of this comment. \u000a
```
may be compiled without error, while we only concern about string literal and char literal in this check (the accepted token types). To sum up, it is impossible.